backend/typescript/services/implementations/camperService.ts

```diff
@@ -135,6 +140,12 @@ class CamperService implements ICamperService {
 
       checkoutSessionUrl = createStripeCheckoutSessionResponse.url;
 
+      // Create new unused refund code
+      do {
+        refundCode = cryptoRandomString({length: REFUND_CODE_LENGTH, type: 'alphanumeric'});
+        campersWithRefundCode = await MgCamper.find({ checkoutSessionUrl });
```

Comment:
perhaps change this to `campersWithSameCode` so that its a little more clear whats happening

Reply:
thank you, I agree! Have made the change in the latest commit.